### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by hsmwrv** [getlantern/lantern-forum-cn#1097] [General]
按FAQs里的方法都试遍了，还是连不上。写了封邮件给support，回复说创建了一个ticket并给了个链接：https://support.lantern.io/helpdesk/tickets/370，但是链接也打不开。紧接着又来一封邮件说什么国际支付没法退款，支付宝的收款方明明就是国内。现在连pro账号也登录不上。

**Comment by hsmwrv** [comment]
pro账号9813255@qq.com
![image](https://github.com/user-attachments/assets/49b32c37-d93a-4118-93d2-fd96136f5223)

**Comment by MaxMKMax** [comment]
@hsmwrv 您好. 现在美国团队在假期. 我会先帮您提交问题上去. 请您耐心等待. 谢谢.
